ui/package.json

```diff
@@ -84,14 +87,35 @@
     "wiredep": "~2.2.0"
   },
   "engines": {
-    "node": ">=6.10.x"
+    "node": ">=6.10.x",
+    "yarn": ">= 1.0.0"
   },
   "scripts": {
     "init": "npm install",
     "install": "bower install",
-    "test": "./node_modules/.bin/karma start --single-run --browsers Chrome"
+    "test": "./node_modules/.bin/karma start --single-run --browsers Chrome",
+    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'bower_components', 'junction') } catch (e) { }\""
   },
   "dependencies": {
-    "jasmine-core": "^2.5.2"
-  }
+    "jasmine-core": "^2.5.2",
+    angular.module('myApp', [require('angular')]);
```

Comment:
No @pranayrobo, they should be like 
```
"jasmine-core": "^2.5.2",
"angular": "^1.6.7",
```


Just remove all these and install these like  `npm install angular`, they will get added to the `package,json`